packages/ui/atomic/template/scripts/setup-lambda.mjs

```diff
@@ -0,0 +1,19 @@
+import {symlinkSync} from 'node:fs';
+import {dirname, resolve} from 'node:path';
+import {fileURLToPath} from 'node:url';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+function linkSearchTokenLambdaToRoot() {
+  symlinkSync(
+    resolve(__dirname, '..', 'node_modules', '@coveo', 'search-token-lambda'),
+    resolve(__dirname, '..', 'lambda'),
+    'junction'
```

Comment:
This is the magic word that allows it to work on windows
See this SO if you're curious: https://superuser.com/a/1291446